PROTOCOL.md

```diff
@@ -2807,6 +2938,114 @@ nullCount | The number of `null` values for this column
 minValues | A value smaller than all values present in the file for this column
 maxValues | A value larger than all values present in the file for this column
 
+## API Response Format in Delta
+This section discusses the API Response Format in Delta returned by the server. When a table is shared
+as delta format, the actions in the response could be put in a delta log in the local storage on the
+recipient side for the delta library to read data out of it directly, in this way, advanced delta
+features are transparent to delta sharing, and the code duplication for the new delta features won't
```

Comment:
Consider rephrashing: 
This way of sharing makes the delta sharing protocol more transparent and robust in supporting advanced delta feature, and minimizes code duplication.